.github/workflows/elm_tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Elm Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      
+      - name: Setup Elm
+        uses: jorelali/setup-elm@v3
```

Comment:
We have a policy of referring to workflows via their git SHA instead of a version number, as using a version number (tag) is a potential security vulnerability (the tag can be reassigned to a different commit). As this is an external dependency, this is especially important. Could you update?